components/director/cmd/director/main.go

```diff
@@ -133,8 +133,11 @@ func main() {
 
 	mainRouter.HandleFunc(cfg.RuntimeMappingEndpoint, runtimeMappingHandlerFunc)
 
-	log.Infof("Registering Healthz endpoint...")
-	mainRouter.HandleFunc("/healthz", healthz.NewHTTPHandler(log.StandardLogger()))
+	log.Infof("Registering readiness endpoint...")
+	mainRouter.HandleFunc("/ready", healthz.NewReadinessHandler())
```

Comment:
What do you think about `/readyz`?

https://github.com/kubernetes/kubernetes/pull/78458/files